python_modules/libraries/dagster-dbt/dagster_dbt_tests/conftest.py

```diff
@@ -51,12 +53,15 @@ def dbt_python_config_dir():
     scope="session",
     params=[
         "legacy",
-        "pythonic",
-    ],
+        "DbtCliClientResource",
+    ]
+    + (["DbtCli"] if version.parse(dbt.version.__version__) >= version.parse("1.4.0") else []),
 )
 def dbt_cli_resource_factory(request):
-    if request.param == "pythonic":
+    if request.param == "DbtCliClientResource":
         return DbtCliClientResource
+    elif request.param == "DbtCli":
+        return DbtCli
```

Comment:
We should do the version check here, and then call if `pytest.skip()` if it fails. Just so that it's explicit in the test suite that we are omitting tests: https://docs.pytest.org/en/7.1.x/how-to/skipping.html#skipping-test-functions

Reply:
👍